### PR TITLE
add installation and configuration of xdebug

### PR DESCRIPTION
**Pull request by mbrendolini**
added xdebug installation and configuration

### CODE REVIEW

File: vagrant/ansible/include/php.yml [hunk 1]
```diff
@@ -16,4 +16,10 @@
     - "{{ php_ini_settings }}"
 - name: Install composer
   shell:
-    curl -sS https://getcomposer.org/installer | php && mv composer.phar /usr/local/bin/composer
\ No newline at end of file
+    curl -sS https://getcomposer.org/installer | php && mv composer.phar /usr/local/bin/composer
+- name: Install gcc
+  shell:
+    yum install gcc gcc-c++ autoconf automake
+- name: Install Xdebug
```
Comment (gabriperego) [inline vagrant/ansible/include/php.yml]: PHP modules should be installed in the php.yml using this format: *php-pecl-xdebug*. Please move the Debug installation there

**Comment by mbrendolini** [comment]
moved xdebug installation into vars/php.yml

**Comment by gabriperego** [comment]
+1

ping @guglielmobossi @ninjabachelor @sgentilini

**Comment by guglielmobossi** [comment]
+1

File: vagrant/ansible/include/php.yml [hunk 2]
```diff
@@ -16,4 +16,8 @@
     - "{{ php_ini_settings }}"
 - name: Install composer
   shell:
-    curl -sS https://getcomposer.org/installer | php && mv composer.phar /usr/local/bin/composer
\ No newline at end of file
+    curl -sS https://getcomposer.org/installer | php && mv composer.phar /usr/local/bin/composer
+- name: Install gcc
+  shell:
+    yum install gcc gcc-c++ autoconf automake
```
Comment (sgentilini) [inline vagrant/ansible/include/php.yml]: Is this the correct ansible syntax to run yum?
Shouldn't it be  similar to: yum: pkg={{ item }} state=latest enablerepo=remi,remi-php55

Comment (gabriperego) [inline vagrant/ansible/include/php.yml]: You are right @sgentilini even if the command should work without problems.
Please @mbrendolini use the proper syntax and also move it into the basic.yml file (this is a more generic installation)

**Pull request merged**